packages/ui-react/src/components/Select/Select.module.scss

```
@@ -0,0 +1,100 @@
+@use '@jwp/ott-ui-react/src/styles/variables';
+@use '@jwp/ott-ui-react/src/styles/theme';
+@use '@jwp/ott-ui-react/src/styles/mixins/responsive';
+
+$select-border: variables.$gray-lighter;
+$select-arrow: variables.$white;
+$select-focus: theme.$primary-color;
+
+$disabled-bg-color: #eee;
+$disabled-gradient: linear-gradient(to top, #ddd, $disabled-bg-color 33%);
+
+.container {
+  position: relative;
+  display: grid;
+  align-items: center;
+  width: 100%;
+  height: 36px;
+  font-size: 1.25rem;
+  background-color: rgba(0, 0, 0, 0.54);
+  border: 1px solid rgba(255, 255, 255, 0.3);
+  border-radius: 0.25em;
+  cursor: pointer;
+  grid-template-areas: 'select';
+
+  &.error {
+    border-color: theme.$text-field-error-color;
+  }
+
+  &.medium {
+    height: 48px;
+
+    .select {
+      height: 46px;
+      padding: 14px 16px;
+    }
+  }
+
+  // Custom arrow
+  &::after {
+    z-index: 1;
+    width: 10px;
+    height: 5px;
+    margin-right: 9px;
+    background-color: $select-arrow;
+    clip-path: polygon(100% 0%, 0 0%, 50% 100%);
+    //noinspection CssUnknownProperty
+    -webkit-clip-path: polygon(100% 0%, 0 0%, 50% 100%); /* stylelint-disable-line */
+    content: '';
+    justify-self: end;
+  }
+
+  .select {
+    z-index: 1;
+    width: 100%;
+    height: 36px;
+    margin: 0;
+    padding: 2px 8px;
+    color: variables.$white;
+    font-family: inherit;
+    font-weight: theme.$body-font-weight-bold;
+    font-size: 1rem;
+    line-height: 1em;
+    text-overflow: ellipsis;
+    background: none;
+    border: none;
+    outline: none;
+    cursor: inherit;
+    -webkit-appearance: none; /* stylelint-disable-line */
+    appearance: none;
+
+    // Remove IE arrow
+    &::-ms-expand {
+      display: none;
+    }
+  }
+
+  .select,
+  &::after {
+    grid-area: select;
+  }
+
+  & .disabled {
```

```suggestion
  &.disabled {
```
Small syntax correction. But if you didn't encounter any problems with this selector, you can leave this as is

---

Yes, this was wrong indeed. Fixed now!